### PR TITLE
Neuron follow

### PR DESCRIPTION
**Pull request by jean-dfinity**
This PR adds support for the neuron management FOLLOW operation.

### CODE REVIEW

File: src/commands/neuron_manage.rs [hunk 1]
```diff
@@ -358,6 +366,21 @@ pub fn exec(pem: &str, opts: ManageOpts) -> AnyhowResult<Vec<IngressWithRequestI
...
+            let args = Encode!(&ManageNeuron {
+                id,
+                command: Some(Command::Follow(Follow {
+                    topic, // Topic::NeuronManagement as i32,
+                    followees,
+                })),
+                neuron_id_or_subaccount: None,
+            })?;
+            msgs.push(args);
+        }
```
Comment (MarioDfinity) [inline src/commands/neuron_manage.rs:381]: Shouldn't the else be an error? Either the user sets both follow_topic and follow_neurons or not, all the other combinations should be an error to the user.

**Review by MarioDfinity — APPROVED** [review]

File: src/commands/neuron_manage.rs (hunk 1 above)
Comment (jean-dfinity) [reply src/commands/neuron_manage.rs:383]: I think I have to add you specifically to the reviewers list if you're not code owner. Can you try again now?

Comment (jean-dfinity) [reply src/commands/neuron_manage.rs:383]: Damn fast :D

File: src/commands/neuron_manage.rs [hunk 2]
```diff
@@ -358,6 +366,23 @@ pub fn exec(pem: &str, opts: ManageOpts) -> AnyhowResult<Vec<IngressWithRequestI
         msgs.push(args);
     };
 
+    if let Some(topic) = opts.follow_topic {
+        if let Some(neuron_ids) = opts.follow_neurons {
```
Comment (MarioDfinity) [inline src/commands/neuron_manage.rs:370]: Note that technically you need to do also the opposite check. If follow_neurons is set then follow_topic must be set. I let you decide if it's important or not but I would add a check (I would change this to be `match (opts.follow_topic, opts.follow_neurons) {...` and then signal error in all cases except when `(None, None)` and `(Some(..), Some(..)`).

**Review by MarioDfinity — APPROVED** [review]

**Review by chmllr — APPROVED** [review]

**Pull request merged**